config/master/calico.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         # container programs network policy and routes on each
         # host.
         - name: calico-node
-          image: quay.io/calico/node:v3.16.2
+          image: calico/node:v3.16.2
```

Comment:
You will probably want to explicitly call out the registry here. e.g., `docker.io/calico/node:v3.16.2`

Reply:
Yes will do that

Reply:
This is awesome! I spent an hour trying to figure why quay didn't work for graviton, then I saw dockerhub had both arm64 and x86_64 architecture listed for the same tag and decided to try. I have a cluster with both Archs and wanted the daemonset to run on all nodes

I got it working with `helm upgrade --install --set calico.node.image="calico/node" --namespace kube-system aws-calico eks/aws-calico`.

I am going to put a PR for the helm charts!

Reply:
Just ran into this exact same issue. :(